### PR TITLE
.github/workflows/nightly.yml: fix used base image

### PR DESCRIPTION
**Pull request by invidian**
It also still depends on https://github.com/newrelic/infrastructure-bundle/pull/67

Signed-off-by: Mateusz Gozdek <mgozdek@microsoft.com>

### CODE REVIEW

**Review by paologallinaharbur — APPROVED** [review]
LGTM

**Comment by invidian** [comment]
It's passing now with the test image: https://github.com/newrelic/nri-kubernetes/pull/144/checks?check_run_id=2971347145.

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ on:
     branches:
       - master
       - main
+      - invidian/fix-nightlies
```
Comment (paologallinaharbur) [inline .github/workflows/nightly.yml:10]: don't lie, it is an awesome to finally run ci before merging

**Comment by invidian** [comment]
This is now also passing.

**Review by paologallinaharbur — APPROVED** [review]

**Pull request merged**